packages/react/src/components/CodeMirror6.jsx

```diff
@@ -140,6 +142,12 @@ export default function CodeMirror({
     }
     return staticExtensions;
   }, [keybindings]);
+
+  const setAutoCompletion = (isAutoCompletionEnabled) =>
```

Comment:
why is this a function? couldn't this be added to extensions inside the useMemo (L135+) ?
outside of the memo the function will be created and the extensions will be spreaded on every render.
This creates a little bit of garbage on every keystroke

Reply:
Indeed, fixed